data-types/collections/spec/hashes_spec.rb

```diff
@@ -127,7 +138,9 @@
     # add a new key of :Ernest_Hemmingway
     # with a value of "For Whom the Bell Tolls"
     # in the line below
-    ______
+
+    books={   :John_Steinbeck => "Grapes of Wrath",
+      :Harper_Lee => "To Kill a Mockingbird",:Ernest_Hemmingway => "For Whom the Bell Tolls"}
```

Comment:
You are doing wrong here. Question asks you to add a new key to the books hash not create a new hash altogether.